backend/chalice/api_server/app.py

```diff
@@ -48,7 +48,9 @@ def get_chalice_app(flask_app):
     app.log.setLevel(logging.DEBUG)
 
     def dispatch(*args, **kwargs):
-        app.log.info(f"Request: {app.current_request.to_dict()}")
+        log_request = app.current_request.to_dict()
```

Comment:
Very minor nit: just to avoid duplicate code, you could create a tiny helper function like `clean_entry_for_logging` that does both the dictionary conversion and the removal of the body and use it in both locations.

Reply:
Done